code/game/objects/structures/crates_lockers/crates/syndicrate.dm

```diff
@@ -0,0 +1,88 @@
+/obj/structure/closet/crate/syndicrate
```

Comment:
This mirror's branch seems out of date, this file was added in #18209 while this PR acts as if it doesn't exist.